src/std/build-std.ss

```diff
@@ -6,5 +6,12 @@
 
 (include "build-spec.ss")
 
+(def (gerbil-build-nproc)
+  (cond
+   ((getenv "GERBIL_BUILD_NPROC" #f)
+    => (lambda (x) (with-catch false (lambda () (def n (string->number x)) (if (< 0 n) n #f)))))
+   (else #t)))
+
 (def (build srcdir libdir)
-  (make build-spec srcdir: srcdir libdir: libdir debug: 'src))
+  (make build-spec srcdir: srcdir libdir: libdir debug: 'src
+        parallelize: (gerbil-build-nproc)))
```

Comment:
I think we should move the call to `(gerbil-build-nproc)` and its definition inside make so that it is used by all projects and not just the stdlib build.